README.md

```diff
@@ -68,6 +68,8 @@ Bookstore is app that I am going to create using react and redux. I will update
 
```

Comment:
![image](https://github.com/Atifzada/BookStore_App/assets/93039359/6a55d652-2b91-4a6a-b158-43a3a4e77bb6)

- [ ] It would be nice if you double-check your LinkedIn and Github links in the readme file to make your readme descriptive and professional 😊 because now they are broken and showing me 404 errors 😨 Kindly fix these errors to make yourself easily accessible by the viewers of your repository 👍